### DISCUSSION TITLE
How to limit to daily and weekly backups

### DISCUSSION THREAD

**Discussion by fizcris** [prodrigestivill/docker-postgres-backup-local#104] [Q&A] [answered]
Hi and sorry for the noob question.

 I wanted to keep only daily and weekly backups and **disable monthly backups**  (to save a bit of HD space) but even if you do `BACKUP_KEEP_MONTHS=0` it still does a monthly backup.... 

So, it´s there a way to disable monthly backups or the minimum allowed is to have at least one daily, weekly and monthly backup? 

Cheers

**Comment by prodrigestivill** [comment, marked as answer]
The backup folder structure is using hard links.
So all files from the same date are the same and they don't use extra disk usage.

So even that you see the backup file is also in the folder monthy folder, it is not consuming disk usage.
In any case if you set BACKUP_KEEP_MONTHS=0 it will not consume disk usage for monthly backups, even that you will see the files there.

More info: https://en.wikipedia.org/wiki/Hard_link
